client/components/zendesk-chat/index.jsx

```diff
@@ -0,0 +1,18 @@
+import { useEffect } from 'react';
+
+const ZendeskChat = ( { chatId } ) => {
+	useEffect( () => {
+		if ( chatId ) {
```

Comment:
Let's maybe return early if `chatId` is false-y to avoid nesting.
```suggestion
		if ( ! chatId ) {
			return;
		}
```